vpr/src/place/median_move_generator.cpp

```diff
@@ -5,9 +5,9 @@
 #include "placer_globals.h"
 #include "move_utils.h"
 
-static bool get_bb_incrementally(ClusterNetId net_id, t_bb* bb_coord_new, int xold, int yold, int xnew, int ynew);
+static bool get_bb_incrementally(ClusterNetId net_id, t_bb& bb_coord_new, int xold, int yold, int xnew, int ynew, int layer);
```

Comment:
The function in "place.cpp" that updates the bounding box incrementally is named "update_bb". We cannot reuse this function because it checks specific static variables within the "place.cpp" file to determine whether the bounding box has already been updated or not.